docs/source/datasets/index.rst

```diff
@@ -172,6 +172,63 @@ about CPS, please visit the `CPS homepage <https://www.census.gov/programs-surve
 
 WIC
 ---
+The Special Supplemental Nutrition Program for Women, Infants, and Children (WIC) is a government benefits program designed to support mothers and young
+children. The main qualifications are income and the presence of young children in the home. To find out more about this service, please visit the `WIC 
+homepage <https://www.fns.usda.gov/wic>`_.
+
+.. list-table:: **Simulant attributes**
+   :header-rows: 1
+
+   * - Attribute Name
+     - Column Name
+     - Notes
+   * - Unique simulant ID
+     - simulant_id
+     - Not affected by noise functions; intended use is 'ground truth' for PRL tracking. 
+   * - Household ID 
+     - :code:`household_id` 
+     - Not affected by noise functions; intended use is 'ground truth' for PRL tracking.
+   * - First name
+     - :code:`first_name`
+     - 
+   * - Middle initial
+     - :code:`middle_initial`
+     - 
+   * - Last name
+     - :code:`last_name`
+     - 
+   * - Age
+     - :code:`age`  
+     - Rounded down to an integer.
+   * - Date of birth
+     - :code:`date_of_birth`
+     - Formatted as MM/DD/YYYY.
```

Comment:
```suggestion
     - Formatted as MMDDYYYY.
```

It should be MMDDYYYY according to the concept model doc. Can you double-check in the actual pseudopeople output?

Reply:
Oh in the pseudopeople output it is looking like YYYY-MM-DD actually. Does that seem okay?

Reply:
Although I am not looking at the most up-to-date output, because I didn't know how to load the parquet files. Lmk if it's worth it to check that, otherwise if someone else could check that has been able to load the new results - that would be great